CHANGELOG.md

```diff
@@ -1,3 +1,58 @@
+## 2023-07-26 v0.19.0
+
+## What's Changed
+
+Release highlights:
+* Postgres support has been added as a backend for archive module
+* RLN initialization optimizations
+* Update to the latest nim-libp2p
```

Comment:
Perhaps a more precise highlight here would be: improved connection management, including management for non-relay peers and limiting the number of connections from a single IP

Reply:
Thanks! Added!